.dotnet/tests/Images/ImageGenerationTests.cs

```diff
@@ -17,10 +17,22 @@ namespace OpenAI.Tests.Images;
 [Category("Images")]
 public partial class ImageGenerationTests : SyncAsyncTestBase
 {
+    private const string CatPrompt = "A big cat with big round eyes and large cat ears, sitting in an empty room and looking at the camera.";
```

Comment:
Storing this in a const var to make it easier to update the prompt later. Unfortunately we may need to think about a better prompt for validation because this one is a bit flaky. Sometimes they see a raccoon, sometimes an owl, sometimes just an "animal".